clang/test/dpct/asm/prefetch_default.cu

```diff
@@ -0,0 +1,70 @@
+// UNSUPPORTED: cuda-8.0
```

Comment:
This lit test could be removed; For the not-supported scenarios, the current ASM migration framework will emit a warning message for the "SYCLGenError()" scenario.